development-vm/README.md

```diff
@@ -1,276 +1,212 @@
 
-* Your VM comes pre-configured with an IP address. This is visible in
-  the [Vagrantfile](./Vagrantfile) (but currently defaults to `10.1.1.254`)
-* If you have < 8GB of RAM on your host machine, you will need to reduce the
-  RAM available to the VM. You can also add extra RAM if you require more.
-  You can do both of these things in a `Vagrantfile.localconfig` file in this
-  directory, which is automatically read by Vagrant (don't forget to run
-  `vagrant reload`!):
+To create a new account, start by creating an SSH key of sufficient key strength.
 
-        $ cat ./Vagrantfile.localconfig
-        config.vm.provider :virtualbox do |vm|
-          vm.customize [ "modifyvm", :id, "--memory", "1024", "--cpus", "2" ]
-        end
+    mac$ ssh-keygen -t rsa -b 4096 -C "your_email@example.com" -f ~/.ssh/alphagov
 
-* If you want to be able to SSH into your VM directly, consider using `vagrant
-  ssh` as it'll always do the right thing. If you need direct access (for
-  `rsync`, `scp` or similar), you'll need to manually configure your ssh
-  configuration: run `vagrant ssh-config --host dev` and paste the output into
-  your `~/.ssh/config`. It'll now be available to SSH into using `ssh dev`.
+Now create a user manifest in `~/govuk/govuk-puppet/modules/users/manifests` with your username and the public key we just created.
 
-* To re-run Puppet, just SSH into your VM and run `govuk_puppet`.
+Add the name of your manifest (your username) into the list of `users::usernames` in `hieradata/integration.yaml`.
 
-* If for example you want to install ZSH, your default root password is `vagrant`.
-  (https://www.vagrantup.com/docs/boxes/base.html#root-password-quot-vagrant-quot)
+There's another bit of hieradata in `integration.yaml` for `govuk_jenkins::config::admins` which will give you access to the deployment Jenkins. Add your GitHub Enterprise username (usually your full name) to this list.
 
-## Troubleshooting
+Create a pull request with these changes. Once it has been reviewed by a member of the gov.uk team, you can merge it and it will automatically deploy to the integration environment. (This will come in handy in later steps.)
 
-### Errors loading the Vagrantfile
+## 4. Boot your VM
 
-If you're encountering errors loading the `Vagrantfile`, check you're running the right version:
+Now you can run the VM bootstrap script:
 
-    vagrant --version
+    mac$ cd govuk-puppet/development-vm
+    mac$ ./bootstrap
 
-if it reports a version below `1.3.x`, you're out of date. This can be
-verified by running `which rbenv`, which will likely report a path
-like `/opt/boxen/rbenv/shims/vagrant`.
+This will take a little while, but it will throw up a question or two in your console so check back on it occassionally. Now might be a good time to scan through the [gov.uk technology blog](https://gdstechnology.blog.gov.uk/category/gov-uk/) while puppet runs.
```

Comment:
[We refer to 'puppet' and 'Puppet' in this guide. Again, for consistency, we should stick with one or the other.]

Also, change gov.uk to GOV.UK.